public/doc/swagger-2-v0.1.0.yaml

```diff
@@ -1354,7 +1354,7 @@ definitions:
         type: string
         example: Endpoint
       resource_id:
-        "$ref": "#/definitions/ID"
+        "$ref": "#/definitions/UpdateableID"
```

Comment:
I think this can change to `ID` as well, then we don't need `UpdateableID`

Reply:
Ah ok, sounds good, it was the only one that wasn't originally set to `IDReadOnly` so I kept the consistency. We can always re-add `UpdateableID` if this or some other relational object id should be updateable.